HeatApp/components/screens/UserSettingsScreen.js

```diff
@@ -14,27 +16,202 @@ export default class UserSettingsScreen extends React.Component {
     )
   };
 
+  genderMatch = {
+    1: 'Male',
+    2: 'Female',
+    3: 'Other'
+  };
+
   constructor(props) {
     super(props);
+    this.state = {
+      gender: 3,
+    };
+  }
+
+  getStateClone() {
+    var stateClone = {};
```

Comment:
I think it's better to use `let` and `const` instead of `var` (everywhere, not only there).
(https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e)